tools/aws_benchmarking/pserver.sh.template

```diff
@@ -0,0 +1 @@
+nvidia-docker run -i -e "TRAINING_ROLE=PSERVER"  -e "PSERVER_HOSTS={PSERVER_HOSTS}" {DOCKER_IMAGE}
```

Comment:
May we need a argument `-p` to port the container port to host.

Reply:
thanks, will add it